packages/donate-button-beta/src/public/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <title>Every Month Widget</title>
+  </head>
+
+  <body
+    style="background-size: cover; background-repeat: no-repeat; overflow: auto"
+  >
+  <div id="button-root" />
+    <script>
+      window.addEventListener("load", function() {
+        everyDotOrgDonateButton.createButton({
+          selector: "#button-root",
+          nonprofitSlug: "everydotorg",
+          crypto: true
+        });
+      })
```

Comment:
Very basic example of what an embed would look like; the ultimate embed script would be slightly more than this:

```html
    <div id="edo-donate-btn" />
    <script src="path/to/the/script.js" class="edo-donate-button-script"></script>
    <script>
      (() => {
        const edoScript = document.querySelector(".edo-donate-button-script")
        if (!edoScript) return;
        edoScript.onload = () => {
          everyDotOrgDonateButton.createButton({
            selector: "#edo-donate-btn",
            nonprofitSlug: "everydotorg",
            crypto: true
          });
        })
      )();
    </script>
```

Reply:
LGTM! I think this is fine for getting something up and running quickly, but I also think we can learn from TikTok here is what they do

```
  <blockquote
    class="tiktok-embed"
    cite="https://www.tiktok.com/@kyliejenner/video/6951976455400099077"
    data-video-id="6951976455400099077"
    style="max-width: 605px; min-width: 325px"
  >
    <section>
      <a
        target="_blank"
        title="@kyliejenner"
        href="https://www.tiktok.com/@kyliejenner"
        >@kyliejenner</a
      >
      <p>time goes too fast 😥😥🤍</p>
      <a
        target="_blank"
        title="♬ original sound - sam aldridge"
        href="https://www.tiktok.com/music/original-sound-6750691377606216453"
        >♬ original sound - sam aldridge</a
      >
    </section>
  </blockquote>
  <script async src="https://www.tiktok.com/embed.js"></script>
```

Several nice things about this:
- Script is async
- Without Javascript, instantly has multiple links for PageRank - the video, the person, the music. In our case, would be great to instantly have href to the nonprofit's main page (not their donate crypto page) for pagerank purposes.
- Devs instantly know how to adjust the width without reading any documentation
- Metadata included entirely via html, makes it look a bit cleaner and more obvious what to if you have multiple embeds
- If client does not have Javascript, at least they can still get to the persons profile. In our case, would be nice if they can still get to the /donate/crypto page (though admittedly that page would then fail, but perhaps someone would be willing to explicitly enable javascript for www.every.org and not less well-known domains)

Reply:
Yeah the way I wrote the code the script can be async, I just forgot. Re the richer content I'll change it to by default not take a selector for where to make the element but attach a handler directly to a donate crypto link. Don't think we can add a whole lot more metadata though since this is supposed to be a single button, not a whole tweet or Tiktok post or something.

Reply:
We can trim down the copy paste code to remove needing to explicitly call `initButtons()` as well by automatically calling `initButtons()` in the downloaded script - im not in love with the automatic side effects, but it is simplest. that said if people just copy-paste this it's not that bad as is (esp if minified)

Reply:
styling will not be as direct as just styling the underlying elemetns though, since they're in a Shadow DOM to provide encapsulation - but it's not any worse than tik tok or twitter, who load their stuff in `iframe`s, which prevent meddling just as much.